VogtPayroll4/EmployeePayrollConsoleReader.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.IO;
+
+namespace VogtPayroll4
+{
+    class EmployeePayrollConsoleReader
+    {
+        public List<Employee> CreateNewEmployee()
```

Comment:
The name doesn't match what is being done. ReadEmployeesFromFile would be more descriptive.